integrations/pinterest-tag/test/index.test.js

```diff
@@ -9,6 +9,7 @@ var Pinterest = require('../lib/');
 describe('Pinterest', function() {
   var analytics;
   var pinterest;
+  var messageId;
```

Comment:
Could you remove the test you wrote for previously? I don't think it is applicable anymore. Is it possible to write a new test that just validations if event_id is set or not? We need not validate the value itself but presence of any value should be fine. This is to ensure that if condition works,